docs/application/dotnet/guides/nui/nui-components/Pagination.md

```diff
@@ -4,135 +4,222 @@ Pagination is a common component supporting multiple pages. It shows the number
 
 ![Pagination](./media/Pagination.png)
 
+
+In the following sections on this guide one can find:
+- [the main Pagination properties](#paginationProperties)
+- how to create a Pagination
+   - [using Properties](#createWithProperty)
+   - [using Style](#createWithStyle)
+   - [using a custom style](#createWithCustomStyle)
+- [how to connect Pagination with a KeyEvent](#connectWithKeyEvent)
+
+
```

Comment:
in sections like:
https://docs.tizen.org/application/dotnet/guides/nui/text/
or
https://docs.tizen.org/application/dotnet/guides/nui/visuals/
we have such an agenda, but it is not named this way. I think this makes it easier for the reader to navigate around the site, but if you confirm, I will of course remove these lines from the guide

Reply:
What my intention was that all pages under 
tizen-docs/docs/application/dotnet/guides/nui/nui-components/

do not have such structure, since I do agree with your reasoning, but to maintain consistency, we may remove those lines, but yes I would like @annie-samsung to give her valuable suggestion here.

Reply:
I haven't made this change yet. Please note that a similar introduction is given in the chapter on Visuals.
I will delete this fragment if you confirm that we don't want it

Reply:
thank you for these comments.
I removed this part and I will also remove it from 'visuals' guide.